packages/cli/src/cmds/validator/signers/importExternalKeystores.ts

```diff
@@ -55,8 +55,6 @@ export function isVotingKeystore(filename: string): boolean {
     //       However, that's too restrictive. Guide left here as a reference
     //
     // The format exported by the `staking-deposit-cli` library.
-    //
-    // Reference to function that generates keystores:
```

Comment:
Why was this removed?  No changes should be included in this PR other than the links themselves...